Containerfile.build

```diff
@@ -0,0 +1,21 @@
+FROM docker.io/library/alpine:latest
```

Comment:
I forgot that one reason I made a separate container for building is to keep the compiler out of the final image. Some gems require a compiler. We'll have to retain the build container. We can base it on `docker.io/library/ruby:alpine` though. The final image will still be based on `docker.io/library/alpine:latest` because of this.

Reply:
In the end, both Containerfiles were rebased on `docker.io/library/alpine:latest`.